src/main/io/rc_controls.h

```diff
@@ -249,6 +257,8 @@ typedef struct adjustmentRange_s {
 
     // ... via slot
     uint8_t adjustmentIndex;
+    uint8_t enableDirectMode;
```

Comment:
Hmm, I'm not sure what the capacity of the EEPROM is.
From the looks of it, this diff adds:
Per adjustment range:
1 x `uint8_t`
2 x `float`
So (1+8) * 12 adjustment ranges = 108 bytes.